### ISSUE TITLE
Linux file doesn't run on Raspberry Pi

### ISSUE THREAD

**Issue by leecollingsco** [swift-project/pilotclient#214]
After downloading the .run file (designated for Linux), when trying to run it on my Raspberry Pi, the system doesn't know what to do with it, and is asking for an application to open the file.

What application should I use to run it?

**Comment by ltoenning** [comment]
Normally you can run it from the terminal by setting the executable bit and launching the run file.

**BUT:** Raspberry Pi uses an ARM processor. Currently we only support x86 and x64 and hence it will probably not work. You could try to compile swift yourself. IIRC some users on the swift discord server already tried this but did not succeeded. Even if you could compile swift yourself, you wouldn't be able to connect to the  VATSIM servers.

Long story short: Unfortunately there is no easy and good way to run swift on a RPi right now.

Out of interest: Are you running the flight simulator on the RPi? If yes, which simulator? Flightgear? XP?
